docs/developers/surf-testmachine-setup.md

```diff
@@ -32,15 +32,16 @@ where `surf-windows-ddev` is the name of the job on the windows machine (it's fr
 
 1. Install [homebrew](https://brew.sh/)
 2. Install golang/git/docker with `brew install golang git docker`
```

Comment:
Do we really need golang? I thought we did all this in a build container. it doesn't hurt, of course.

Reply:
Thanks - We do testing right on the container, because of the container-in-container issue (for Circle). But we use the same `make test` override here.